### ISSUE TITLE
Robot should warn, not throw error, when extract seed does not contain ontology terms

### ISSUE THREAD

**Issue by matentzn** [ontodev/robot#434]
I have never noticed that before for some reason (@cmungall ODK hid it from me), but currently, when I try to extract a module with a seed that is disjoint with the ontologies signature, ROBOT fails hard. For no reason: it is IMHO perfectly fine to attempt to extract a module with a disjoint or even empty seed; It is formally defined and the OWLAPI extracts it just fine. Could we drop the error level to warning? That would help a lot, for example when seeding a new ODK. (Note that in the example below, CHEBI:0000001 is not in the ontology, CHEBI does not have trailing 0s).

```
/hp/src/ontology master $ sh run.sh robot extract -i mirror/chebi.owl -T imports/chebi_terms_combined.txt --method BOT  annotate --ontology-iri http://purl.obolibrary.org/obo/hp/imports/chebi_import.owl --version-iri http://purl.obolibrary.org/obo/hp/releases/2019-02-04/imports/chebi_import.owl --output imports/chebi_import.owl
EMPTY TERMS ERROR ontology does not contain input terms
For details see: http://robot.obolibrary.org/errors#empty-terms-error
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
/hp/src/ontology master $ cat imports/chebi_
chebi_import.owl          chebi_terms.txt           chebi_terms_combined.txt  
/hp/src/ontology master $ cat imports/chebi_terms_combined.txt 

CHEBI:0000001
CHEBI:0000002
CHEBI:0000003
http://purl.obolibrary.org/obo/HP_00000000
http://purl.obolibrary.org/obo/hp.owl
http://purl.obolibrary.org/obo/hp/imports/chebi_import.owl
http://purl.obolibrary.org/obo/hp/imports/cl_import.owl
http://purl.obolibrary.org/obo/hp/imports/go_import.owl
http://purl.obolibrary.org/obo/hp/imports/hsapdv_import.owl
http://purl.obolibrary.org/obo/hp/imports/mpath_import.owl
http://purl.obolibrary.org/obo/hp/imports/nbo_import.owl
http://purl.obolibrary.org/obo/hp/imports/pato_import.owl
http://purl.obolibrary.org/obo/hp/imports/ro_import.owl
http://purl.obolibrary.org/obo/hp/imports/uberon_import.owl
http://www.w3.org/2002/07/owl#Class
http://www.w3.org/2002/07/owl#Ontology
term
http://purl.obolibrary.org/obo
/hp/src/ontology master $ sh run.sh robot --version
ROBOT version 1.3.0
```

**Comment by jamesaoverton** [comment]
You make good points.

I don't like changing behaviour that somebody might be relying upon, even when I'm convinced that the existing behaviour has problems.

What do you think of adding a `--force true` option that would warn instead of failing?

**Comment by matentzn** [comment]
Works for me :)

**Comment by rctauber** [comment]
@matentzn if you get a chance, can you see if #435 does what you need?

**Comment by matentzn** [comment]
I think so! If 

```
.. tin = CommandLineHelper.getTerms(ioHelper, line);
OntologyHelper.filterExistingTerms(o, tin, force)
```
returns an empty set of IRIs in case there is no overlap between signature(o) and **tin**, this will do it! 👍 
Thanks!

**Comment by rctauber** [comment]
Great, this is resolved by #435

**Comment by matentzn** [comment]
Hey! I am currently trying to embed this feature in the ODK, but I cant get it to work; according to logs, this feature was merged in early February, and the docs say it works; but when I download the source code of the latest release, it does not seem to be in there (the latest release is dated at least 3 weeks after this merge). @rctauber Can you confirm whether or not ROBOT 1.4.0 allows the force true feature?

```
robot -vv extract -i logical_dependencies_merged.owl --term-file src_seed.txt --method BOT --force true -o $@.mod.owl
```

Fails with:

```
EMPTY TERMS ERROR ontology does not contain input terms
For details see: http://robot.obolibrary.org/errors#empty-terms-error
``` 
if src_seed.txt is empty.

**Comment by rctauber** [comment]
Thanks for catching this. The option wasn't being picked up because it wasn't part of the default extract options. I fixed that in #500.

**Comment by matentzn** [comment]
Great, thanks :)
